node-src/lib/checkForUpdates.ts

```diff
@@ -1,15 +1,15 @@
 import semver from 'semver';
 import { hasYarn } from 'yarn-or-npm';
 
-import { InitialContext } from '..';
+import { Context } from '..';
 import outdatedPackage from '../ui/messages/warnings/outdatedPackage';
 import spawn from './spawn';
 
 const rejectIn = (ms: number) => new Promise<any>((_, reject) => setTimeout(reject, ms));
 const withTimeout = <T>(promise: Promise<T>, ms: number): Promise<T> =>
   Promise.race([promise, rejectIn(ms)]);
 
-export default async function checkForUpdates(ctx: InitialContext) {
+export default async function checkForUpdates(ctx: Context) {
```

Comment:
What's this change doing?  Why not use the tighter scoped type?

Reply:
It's because we're using `ctx.options` which doesn't exist on `InitialContext`!
![image](https://github.com/user-attachments/assets/5d82b4d3-4e37-495f-9089-27e76948dc27)